### PR TITLE
Fix stack traces for required modules

### PR DESCRIPTION
**Pull request by seishun**
Before:
```
error: Uncaught Error: illegal token '105' (../Protobufs/steam/steammessages_clientserver_login.proto, line 1)
► <unknown>:94:16
    at illegal (<unknown>:94:16)
    at parse (<unknown>:734:23)
    at process (<unknown>:108:30)
    at <unknown>:183:17
    at fetchReadFileCallback (<unknown>:51:19)
    at file:///C:/Users/Nikolai/deno/std/node/_utils.ts:33:26
```
After:
```
error: Uncaught Error: illegal token '105' (../Protobufs/steam/steammessages_clientserver_login.proto, line 1)
► file:///C:/Users/Nikolai/deno-steam/node_modules/protobufjs/src/parse.js:94:16

94         return Error("illegal " + (name || "token") + " '" + token + "' (" + (filename ? filename + ", " : "") + "line " + tn.line + ")");
                  ^

    at illegal (file:///C:/Users/Nikolai/deno-steam/node_modules/protobufjs/src/parse.js:94:16)
    at parse (file:///C:/Users/Nikolai/deno-steam/node_modules/protobufjs/src/parse.js:734:23)
    at process (file:///C:/Users/Nikolai/deno-steam/node_modules/protobufjs/src/root.js:108:30)
    at file:///C:/Users/Nikolai/deno-steam/node_modules/protobufjs/src/root.js:183:17
    at fetchReadFileCallback (file:///C:/Users/Nikolai/deno-steam/node_modules/@protobufjs/fetch/index.js:51:19)
    at file:///C:/Users/Nikolai/deno/std/node/_utils.ts:33:26
```
I'm quite new to Rust, so any suggestions on the code will be greatly appreciated.

### CODE REVIEW

**Comment by ry** [comment]
@seishun Hey thanks - looks fine to me. But would you be able to add a test that shows the new output? Follow the example of 001_hello.js in cli/tests/integration_tests.rs

**Comment by seishun** [comment]
Is cli/tests/integration_tests.rs the right place for a test for an std module?

**Comment by ry** [comment]
@seishun I guess the test could use std... but ideally it will be just a small standalone script.

**Comment by seishun** [comment]
Added a test.

**Comment by seishun** [comment]
I can't reproduce the test failure locally. Seems unrelated.

**Comment by bartlomieju** [comment]
@seishun can you rebase this PR to latest master?

**Comment by seishun** [comment]
@bartlomieju I've rebased it, but it seems it wasn't necessary (no conflicts).

File: cli/js/globals.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ declare global {
     shared: SharedArrayBuffer;
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    evalContext(code: string): [any, EvalErrorInfo | null];
+    evalContext(code: string, name?: string): [any, EvalErrorInfo | null];
```
Comment (bartlomieju) [inline cli/js/globals.ts]: nitpick: rename to `filename`

**Comment by ry** [comment]
I have checked that the test fails before the change. I'm +1 on this.

File: cli/js/globals.ts (hunk 1 above)
Comment (ry) [reply cli/js/globals.ts]: renamed scriptName

**Comment by ry** [comment]
@seishun thanks!

**Pull request merged**